pages/docs/react/latest/beyond-jsx.mdx

```diff
@@ -38,9 +38,24 @@ The types above are pretty low level (basically the JS representation of a React
 
 A ReScript React component needs to be a (sub-)module with a `make` function and `props` type to be usable in JSX. To make things easier, we provide a `@react.component` decorator to create those functions for you:
 
-<CodeTab labels={["Decorated", "Expanded"]}>
+<div className="hidden">
```

Comment:
Why hidden?

Reply:
to be able to compile the code in the tests and still don't show the prelude to the user